Products/ZenUtils/controlplane/application.py

```diff
@@ -418,6 +418,8 @@ class _DeployedAppConfigList(Sequence):
 
     def __init__(self, service, client):
         self._service = service
+        if not service._data.has_key("ConfigFiles"):
+            service._data["ConfigFiles"] = client.getService(service.id)._data["ConfigFiles"]
```

Comment:
theres an endpoint for grabbing service configs that will be faster `/api/v2/services/:serviceId/serviceconfigs`